lib/docassemble/docassemble_api_interface.js

```diff
@@ -61,6 +61,10 @@ const time = require('../utils/time');
 let da_i = {};
 module.exports = da_i;
 
+const artifacts_path = session_vars.get_artifacts_path_name();
+const log = new Log({ path: artifacts_path, context: `api` });
```

Comment:
I think I understand what you mean and I like it. Though I wish I could make it all caps so it could look like a proper constant. That would somewhat defeat the purpose, though.